frontend/public/components/catalog-items.jsx

```diff
@@ -294,6 +305,7 @@ export class CatalogTileViewPage extends React.Component {
     const filters = _.cloneDeep(this.state.filters);
     filters.byName.active = filters.byType.clusterServiceClass.active = filters.byType.imageStream.active = false;
     filters.byName.value = '';
+    this.filterByNameInput.focus();
```

Comment:
Was the `autofocus` attribute not working?

Reply:
This is to re-focus on the input field after clearing filters.